charts/zookeeper-operator/templates/pre-delete-hooks.yaml

```diff
@@ -18,7 +22,11 @@ rules:
 ---
 
 kind: ClusterRoleBinding
+{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" }}
```

Comment:
The apiVersion for the `ClusterRole` and `ClusterRoleBinding`  can be changed to `rbac.authorization.k8s.io/v1` and we can remove the check from there, since we are anyways using that version in the other files.

Reply:
I believe this would affect the compatibility with k8s versions 1.15.0 and below, perhaps the other files should also have the check?

Reply:
In our README, we have mentioned requirement as ` Kubernetes v1.15.0+`

Reply:
well I guess for someone running k8s 1.15.0, this is the least of the problem, I'll change the commit removing the check

Reply:
If the requirement k8s v1.15.0+ we should probably check in the other files as well, I can change this, also I double checked, in fact the rbac.authorization.k8s.io/v1beta1 was only deprecated on 1.17.0, so I assume rbac.authorization.k8s.io/v1 is available only from 1.17.0+

the other option would be to update the requirements

Reply:
have  seen that `rbac.authorization.k8s.io/v1` is supported in cluster with version  `v1.15.10`